crates/sncast/src/starknet_commands/account/create.rs

```diff
@@ -106,11 +109,14 @@ pub async fn create(
         )?;
     } else {
         write_account_to_accounts_file(account, accounts_file, chain_id, account_json.clone())?;
+
+        let deploy_command = generate_deploy_command(accounts_file, &rpc_url, account);
+        message.push_str(&deploy_command);
```

Comment:
Why aren't we adding this if keystore is used? If the command would have to be different, I'd either resolve that or add a follow-up issue.

Reply:
Added